packages/webview_flutter/webview_flutter_wkwebview/pigeons/web_kit.dart

```diff
@@ -364,10 +375,12 @@ abstract class WKWebViewHostApi {
 
   void setNavigationDelegate(int instanceId, int? navigationDelegateInstanceId);
 
+  @ObjCSelector('urlForWebViewWithInstanceId:')
```

Comment:
As much as it pains me, the correct naming here is `URLFor...` even though it's the start of the method. See https://developer.apple.com/documentation/webkit/wkwebview/1415005-url?language=objc for instance.

Reply:
maybe `getUrlForWebViewWithInstanceId:`

Reply:
As Maurice mentioned in another comment where I forget, ObjC getters shouldn't actually start with `get`.